reference/test/matrix/csr_kernels.cpp

```diff
@@ -343,6 +343,19 @@ class Csr : public ::testing::Test {
 };
 
 
+TEST_F(Csr, CopyEmptyMatrix)
+{
+    auto strategy = std::make_shared<Mtx::load_balance>(0);
+    auto from_mtx = gko::matrix::Dense<>::create(exec, gko::dim<2>(0, 0));
+    auto to_mtx =
+        gko::matrix::Csr<>::create(exec, gko::dim<2>{0, 0}, 0, strategy);
+
+    from_mtx->convert_to(to_mtx.get());
+
+    ASSERT_FALSE(to_mtx->get_size());
```

Comment:
Explicit comparison might be more expressive
```suggestion
    ASSERT_EQ(to_mtx->get_size(), 0);
```

Reply:
What I am using here is the [`operator bool`](https://github.com/ginkgo-project/ginkgo/blob/develop/include/ginkgo/core/base/dim.hpp#L120). I don't think we define the `operator==` for plain `index_type` (if it would work, then also only because of implicit casts).

Reply:
I see, thanks for the clarification. The name is a bit unintuitive though. I would have expected something like `get_dimensions`.

Reply:
I think it makes sense that the size of a matrix is 2D, but I can definitively understand your confusion.